dts/arm/st/l4/stm32l4r9.dtsi

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2018 Pushpal Sidhu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/l4/stm32l4.dtsi>
```

Comment:
Why not including stm32l4r5.dtsi?
From what I could see l4r9 is a l4r5 + DSI, LTDC and GFX

Reply:
> Why not including stm32l4r5.dtsi?
> From what I could see l4r9 is a l4r5 + DSI, LTDC and GFX

Seems quite good idea

Reply:
@erwango 
Why not adding a stm32l4rx.dtsi equal to the current stm32l4rx.dtsi and then include it in both stm32l4r5.dtsi and stm32l4r9.dtsi ?

What do you think about that? Would that be more clear?

Reply:
@avisconti, we currently apply the same model for all stm32 series with a stm32xx.dtsi base, which is the "root" of the family (peripherals included in all variants of the series). Then this one is included the most simple SoCs variants which are then included in more complex SoCs and this goes on...
Your proposal is adding a second "root" to the stm32l4 series, which is a divergence from this model. Even if i seems fine, I'd prefer we stick to the initial scheme, so we don't see "root" files poping up in various places.

Reply:
That's fine for me. I understand your point.